awx_collection/README.md

```diff
@@ -1,7 +1,18 @@
 # AWX Ansible Collection
 
-This Ansible collection allows for easy interaction with an AWX or Ansible Tower
-server via Ansible playbooks.
+[comment]: # (*******************************************************)
+[comment]: # (*                                                     *)
+[comment]: # (*             WARNING                                 *)
+[comment]: # (*                                                     *)
+[comment]: # (*  This file is manageed by Ansible and not to be     *)
+[comment]: # (*  edited directly! Instead modify:                   *)
+[comment]: # (*  tools/roles/template_galaxy/templates/README.md.j2 *)
+[comment]: # (*                                                     *)
+[comment]: # (*  Changed to the base README.md file are lost upon   *)
+[comment]: # (*  the build of the collection                        *)
+[comment]: # (*******************************************************)
+
+This Ansible collection allows for easy interaction with an AWX server via Ansible playbooks.
```

Comment:
my only real comment here - could you instead just add this to the main `.gitignore`?

Otherwise, commits will be introducing the same diff to 2 files, and that's going to introduce synchronicity issues, which starts going wrong the second that another person contributes.

Reply:
We're kind of in a catch 22 because we want the README file in the repo for AWX but we want a different README to be generated dynamically for ansible.tower. I don't think .gitignore will work here because that only allows you to ignore files that aren't in the index. Since README.md is already there it will still show up in a git status on modification and will still be eligible for commits.

Reply:
if the problem is that the file is in the index, then can't you just delete it and then add it to the `.gitignore`?

> we want the README file in the repo for AWX

Why? Maybe it's nice to have, but I don't see that as necessary. If general direction is needed, it could be put in the [overall project README.md](https://github.com/ansible/awx/blob/devel/README.md)

The main service this README offers is the Galaxy page, and we already kind of preference that formatting over Github.